docs/pages/2019_MARVEL_Psik_MaX/sections/calculations.rst

```diff
@@ -536,4 +512,17 @@ calculation.
 .. rubric:: Footnotes
 
 .. [#f1] However, to avoid duplication of KpointsData, you should first learn how to query the database, therefore we will ignore this duplication issue for now.
-.. [#f2] A process is considered active if it is either ``Created``, ``Running`` or ``Waiting``. If a process is no longer active, but terminatd, it will have a state ``Finished``, ``Killed`` or ``Excepted``.
\ No newline at end of file
+.. [#f2] A process is considered active if it is either ``Created``, ``Running`` or ``Waiting``. If a process is no longer active, but terminatd, it will have a state ``Finished``, ``Killed`` or ``Excepted``.
+
+
+Once run, AiiDA calculations are instances of the class ``CalcJob``,
+more precisely of one of its subclasses, each corresponding to a code
+specific plugin (for example, the PWscf plugin). You have already seen
+``CalcJob`` classes in the previous sections.
+
+However, to create a new calculation, rather than manually creating a
```

Comment:
I personally find this whole section a bit confusing. I think I understand what you are getting at, trying to explain why to use a `ProcessBuilder` instance to launch a `CalcJob` instead of instantiating the `CalcJob` itself. However, saying in the first paragraph  "Once run, AiiDA calculations are instances of the class ``CalcJob``", although true, the thing that users will interact with is the `CalcJobNode`. This explanation may confuse them if this distinction is explained or becomes clear later on. Moreover "However, to create a new calculation, rather than manually creating a new class" is not correct. At least the wording, "create a new class". Maybe you meant "instantiate a new instance of the `CalcJob` class".

Maybe we could rewrite it slightly and link to the documentation I wrote, which should cover all the details.

Any `CalcJob` can be "launched" using one of available launcher functions:
* run
* run_get_node
* run_get_pk
* submit

from the `aiida.engine` module, by passing the calculation class as the first argument followed by the inputs as keyword arguments, e.g.:
```
run(PwCalculation, structure=structure, pseudos={'Si': pseudo_node}, ....)
```
The builder concept is a helpful tool on top of this, to provide tab-completion in the shell and automatic help strings.
Instead of passing the class and inputs manually to the launch function, you can just pass the builder instead.
The launching of calculation jobs is identical for all other processes in AiiDA, such as work chains, which you will encounter later. For more details on launching processes, refer to the [online documentation](https://aiida-core.readthedocs.io/en/latest/working/processes.html#launching-processes)